addons/website/static/src/client_actions/website_preview/website_preview.js

```diff
@@ -84,6 +84,7 @@ export class WebsitePreview extends Component {
             this.websiteService.currentWebsiteId = null;
             this.websiteService.websiteRootInstance = undefined;
             this.websiteService.pageDocument = null;
+            this.title.setParts({ zopenerp: 'Odoo', action: null });
```

Comment:
I wonder if this commit is actually something we want? I would not mind the title still be prefixed by "Odoo -", actually indicating you are in the backend. Did somebody ask for this to be changed? If yes, then maybe it should be entirely consistent and also fix the facat the favicon of the backend is used? :thinking:

Reply:
Let's let @rdeodoo decide then but for:
- Either we use favicon of the website + no "Odoo -"
- Or we leave as it is right now

But not a mix of both...

Reply:
IMHO it just looks buggy when you are optimizing the SEO of you page, you may wonder why is there an "Odoo -" prefixing the tab/title of your page and how to get rid of it.
![image](https://user-images.githubusercontent.com/30048408/191249181-6416d93b-ac28-43eb-936a-f37389e80be2.png)

I agree it might makes sense to have "Odoo -" prefix to illustrate we are in the backend but:
1. It is not needed, we have the backend navbar, it is already easy to figure
2. It would be coming with the problem that when optimizing SEO, it is not perfect (as explained above)

About the favicon, I don't mind either way really, as long as this prefix is gone, unless someone disagrees.
Let's change the favicon too then I guess :+1:

Reply:
Changed according to that :)
It should be ready for review